source/common/termination_predicate_impl.h

```diff
@@ -5,6 +5,8 @@
 
 #include "nighthawk/common/termination_predicate.h"
 
+#include "external/envoy/source/common/common/assert.h"
```

Comment:
Should we also modify the BUILD file to include the necessary target as a dep?

Reply:
Please correct me if I'm wrong but I believe the dependency already is included in the BUILD file under https://github.com/envoyproxy/nighthawk/blob/main/source/common/BUILD#L153.